tests/distutils/run_all.py

```diff
@@ -83,10 +83,12 @@ def main():
         if active:
             my_print("Consider distutils example:", filename)
 
-            if python_version < "3":
-                if filename == "example_3":
-                    reportSkip("Skipped, only relevant for Python3", ".", filename)
-                    continue
+            if python_version < "3" and filename == "example_3":
+                reportSkip("Skipped, only relevant for Python3", ".", filename)
+                continue
+            if python_version < "3.3" and filename == "nested_namespaces":
```

Comment:
Just a minor quibble, we don't support "3.2" anymore, and therefore "<3" and "<3.3" are really the same thing to us, so you can spare the distinction.

Reply:
Ah, good to know. I will change this.